### DISCUSSION TITLE
想问一下在线版的预计开放时间

### DISCUSSION THREAD

**Discussion by Pragmatism0220** [Equim-chan/mjai-reviewer#84] [Q&A] [answered]
因为自己训练的Mortal太菜了
所以现在一直在用akochan复盘牌谱
所以真的很想知道在线版的Mortal什么时候能开放
我真的从Mortal中学到了很多，虽然akochan也有足够值得我学习的地方，但是心里总觉得怪怪的。网站上说预计一个月可能会重新开放服务，所以特地来这里问一下。
如果有打扰真的很抱歉！

**Comment by Equim-chan** [comment, marked as answer]
按照目前的状况大约一周内吧。

**Reply by Pragmatism0220** [reply]
好的感谢！
